packages/table/src/CheckboxCell/CheckboxCell.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Checkbox from '@leafygreen-ui/checkbox';
+import { CheckboxProps } from '@leafygreen-ui/checkbox/src/types';
+import { cx } from '@leafygreen-ui/emotion';
+
+import { baseStyles, headerStyles } from './Checkbox.styles';
+
+const CheckboxCell = (props: CheckboxProps & { isHeader?: boolean }) => (
+  <div className={cx(baseStyles, { [headerStyles]: props.isHeader })}>
```

Comment:
should this not be wrapped in `<td>` as the outer element?

Reply:
It gets passed a child to our Cell component by default